source/programme-talks.rst

```diff
@@ -0,0 +1,74 @@
+Programme of talks
+==================
+
+At PyCon UK 2015, we had approximately 70 talks.
+
+About 57 of the speakers were male, and the vast majority were of course
+extremely white.
```

Comment:
I agree with the sentiment, but I feel like "extremely" is not the world to use here. Suggests a tone I don't think we want.